### PR TITLE
feat: style individual menu bar items

### PR DESCRIPTION
**Pull request by yuriy-fix**

### CODE REVIEW

File: frontend/demo/component/menubar/menu-bar-custom-theme.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import '../../init';
+
+import '@vaadin/vaadin-menu-bar/vaadin-menu-bar';
```
Comment (jouni) [inline frontend/demo/component/menubar/menu-bar-custom-theme.ts:3]: Shouldn't there be a TypeScript version of this example in the `latest` docs?

File: frontend/demo/component/menubar/menu-bar-custom-theme.ts [hunk 2]
```diff
@@ -1,3 +1,36 @@
...
+  ];
+  // end::snippet[]
+
+  render() {
+    return html`
+      <!-- tag::snippethtml[] -->
+      <vaadin-menu-bar .items="${this.items}"></vaadin-menu-bar>
+      <!-- end::snippethtml[] -->
+    `;
+  }
```
Comment (jouni) [inline frontend/demo/component/menubar/menu-bar-custom-theme.ts:35]: I would extend the snippet to cover the `render()` method as well, makes it clearer IMO.

```suggestion

  render() {
    return html`
      <vaadin-menu-bar .items="${this.items}"></vaadin-menu-bar>
    `;
  }
  // end::snippet[]
```

File: articles/ds/components/menu-bar/index.asciidoc [hunk 3]
```diff
@@ -82,12 +82,12 @@ See <<../../customization/custom-theme#vaadin-component-styles, Vaadin Component
 
 [source,typescript]
 ----
-include::{root}/frontend/demo/component/menubar/menu-bar-custom-theme.ts[preimport,hidden]
+include::{root}/frontend/demo/component/menubar/menu-bar-custom-theme.ts[render,tags=snippet;snippethtml,indent=0,group=TypeScript]
```
Comment (jouni) [inline articles/ds/components/menu-bar/index.asciidoc:85]:
```suggestion
include::{root}/frontend/demo/component/menubar/menu-bar-custom-theme.ts[render,tags=snippet,indent=0,group=TypeScript]
```

File: frontend/demo/component/menubar/menu-bar-custom-theme.ts (hunk 2 above)
Comment (yuriy-fix) [reply frontend/demo/component/menubar/menu-bar-custom-theme.ts:35]: That is aligned with other examples:
https://github.com/vaadin/docs/blob/0655cdd8a8316e9620516b0ac068f0d1b253cbe0/frontend/demo/component/menubar/menu-bar-basic.ts#L47
Would it make sense to update it for other examples as well? 🤔

Comment (jouni) [reply frontend/demo/component/menubar/menu-bar-custom-theme.ts:35]: Yes, please, update the other examples as well. But maybe in another PR?

**Review by jouni — APPROVED** [review]

**Pull request merged**